code/datums/supplypacks/supply.dm

```diff
@@ -143,3 +143,18 @@
 	cost = 15
 	containertype = /obj/structure/closet/crate/large/xion
 	containername = "Cargo Train Trolley Crate"
+
+/datum/supply_pack/explorergear
+	name="explorer gear"
+	contains = list(
```

Comment:
Should have a machete sheath, explorer belt, explorer uniform, and explorer headset as well.

Reply:
done